### DISCUSSION TITLE
I want to make a simple file transfer program

### DISCUSSION THREAD

**Discussion by kaikai-y** [microsoft/msquic#2281] [Q&A] [answered]
hi.
I want to make a simple file transfer program by using "sample.c". 
But I don't know how to use send and receive in here.
Can anybody tell me an example how to send a word (like hello world).

**Comment by nibanks** [comment, marked as answer]
If you look at the sample [here](https://github.com/microsoft/msquic/blob/main/src/tools/sample/sample.c#L210) you can see where the server side assigns the buffer (and length) that is to be sent to the client. You could instead point that to a buffer that contains your word (or file).

**Reply by kaikai-y** [reply]
thank you!
